server/register.py

```diff
@@ -10,7 +10,7 @@ def registerUser(email, password):
 
             if(invalidEmail(email, cursor)):
                 return {"response": "Email address is already in use"}
-            sql = "INSERT INTO Users (location, email, password) VALUES ('', %s, %s);"
```

Comment:
What's happing to location / longitude and latitude?

Reply:
Location not needed here. Location updated on other pages instead.